src/components/GallerySection/GallerySection.jsx

```diff
@@ -0,0 +1,34 @@
+import { Link } from "react-router-dom";
+import Card from "../Card/Card";
+import styles from "./GallerySection.module.css";
+
+const GallerySection = () => {
+    return (
+        <section id="gallery" className={styles.gallery}>
+            <div className="container text-center">
+                <h2 className="display-3 pb-lg-3 text-uppercase">Galeria</h2>
+                <div className="row row-cols-1 row-cols-md-2 row-cols-xl-3 g-4">
+                    <Card
+                        imgSrc="/photos/Street Meeting Poland 2024-139.webp"
+                        imgAlt="Czeno biały nissan"
+                    />
+                    <Card
+                        imgSrc="/photos/Street Meeting Poland 2024-252.webp"
+                        imgAlt="Czeno biały nissan"
```

Comment:
I think there are three different images, so the alt texts should be different too.

Reply:
U are right, i will change it